js/bitfinex2.js

```diff
@@ -709,10 +709,10 @@ module.exports = class bitfinex2 extends Exchange {
                 'deposit': undefined,
                 'withdraw': undefined,
                 'fee': fee,
-                'precision': precision,
+                'precision': parseInt (precision),
                 'limits': {
                     'amount': {
-                        'min': 1 / Math.pow (10, precision),
+                        'min': this.parseNumber (this.parsePrecision ('e-' + precision)),
```

Comment:
@samgermain I think this is wrong why are you concatenating 'e-'? 
It should be only `this.parseNumber (this.parsePrecision (precision))`